File: _code-samples/get-started/php/get-account-info.php
<?php

// Use the Composer autoloader
require __DIR__ . '/vendor/autoload.php';

// Imports
use XRPL_PHP\Client\JsonRpcClient;
use XRPL_PHP\Models\Account\AccountInfoRequest;
use XRPL_PHP\Wallet\Wallet;
use function XRPL_PHP\Sugar\fundWallet;

// Create a client using the Testnet
$client = new JsonRpcClient("https://s.altnet.rippletest.net:51234");

// Create a new wallet
$wallet = Wallet::generate();

// Fund (and activate) the wallet
fundWallet($client, $wallet);

// Create an AccountInfoRequest method
$accountInfoRequest = new AccountInfoRequest(
    account: $wallet->getAddress(),
    ledgerIndex: 'validated'
 );

//  Send the request to the XRPL
$accountInfoResponse = $client->syncRequest($accountInfoRequest);

// Print formatted response
print_r($accountInfoResponse);

File: _code-samples/get-started/ruby/get-account-info.rb
require 'xrpl-ruby'
require 'json'

# Create a client using the Testnet
client = XRPL::Client.new(:testnet)
client.connect!

# Create a new wallet and fund it using the Testnet faucet
funded = XRPL.fund_wallet(client)
wallet = funded[:wallet]

puts "Classic address: #{wallet.classic_address}"

# Look up information about the account on the XRP Ledger
account_info = client.account_info_response(
  account: wallet.classic_address,
  ledger_index: 'validated'
)

puts JSON.pretty_generate(account_info)

client.disconnect
